app/routes.py

```diff
@@ -74,6 +74,11 @@ def search():
         except InvalidToken:
             pass
 
+    feeling_lucky = q.startswith("! ")
```

Comment:
```suggestion
    feeling_lucky = q.startswith('! ')
```

Needs single quotes to match the rest of the python formatting